### ISSUE TITLE
Conjugate complex signal block operations are missing

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1349]
**Reported by christiankral on 10 Nov 2013 19:33 UTC**
For some calculations it is required to process a conjugate complex signal, e.g., for the calculation of complex apparent power. 

 1. A new block for conjugate complex signal calculation shall be introduced
 1. Some blocks of ComplexMath shall have boolean parameters to process conjugate complex inputs
   a. `Modelica.ComplexBlocks.ComplexMath.Product`
   a. `Modelica.ComplexBlocks.ComplexMath.Division`
   a. `Modelica.ComplexBlocks.ComplexMath.Add`
   a. `Modelica.ComplexBlocks.ComplexMath.Feedback`
   a. `Modelica.ComplexBlocks.ComplexMath.Gain`

**Example:** In case of `ComplexMath.Product` two additional parameters 
```
  parameter Boolean useConjugateInput1 = false 
    "If true, input 1 is processed conjugate complex";
  parameter Boolean useConjugateInput2 = false 
    "If true, input 2 is processed conjugate complex";
```
shall be introduced and calculation shall be modified to 
```
  y = (if useConjugateInput1 then Modelica.ComplexMath.conj(u1) else u1)
    * (if useConjugateInput2 then Modelica.ComplexMath.conj(u2) else u2);
``` 

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1349

**Comment by modelica-trac-importer** [comment]
**Comment by christiankral on 10 Nov 2013 20:39 UTC**
After discussions with Anton Haumer we are now planning to introduce optional conjugate complex input handling for **all** blocks of the package `Modelica.ComplexBlocks.ComplexMath`. The implementation will be fully backwards compatible.

**Comment by modelica-trac-importer** [comment]
**Comment by christiankral on 11 Nov 2013 18:55 UTC**
Fixed in 8906df8e652ecf8d5a4e03b00e31d33a48fd03db.

**Comment by modelica-trac-importer** [comment]
**Modified by christiankral on 11 Nov 2013 18:55 UTC**
